sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClient.java

```diff
@@ -1014,34 +1010,26 @@ public String getEntityPath() {
      * @throws UnsupportedOperationException if the receiver was opened in {@link ReceiveMode#RECEIVE_AND_DELETE}
      *     mode.
      * @throws IllegalStateException if the receiver is a session receiver.
-     * @throws IllegalArgumentException if {@link MessageLockToken#getLockToken()} returns an empty value.
+     * @throws IllegalArgumentException if {@code lockToken} is {@code null} or an empty value.
      */
-    public Mono<Instant> renewMessageLock(MessageLockToken lockToken) {
+    public Mono<Instant> renewMessageLock(String lockToken) {
         if (isDisposed.get()) {
             return monoError(logger, new IllegalStateException(
                 String.format(INVALID_OPERATION_DISPOSED_RECEIVER, "renewMessageLock")));
         } else if (Objects.isNull(lockToken)) {
```

Comment:
The check doesn't match the documentation. This throws a NPE rather than IllegalArgumentExceptionwhen lockToken is null.